### PR TITLE
refactor: update rector configuration for php8.1

### PR DESCRIPTION
**Pull request by leMaur**
## Checklist

-   [ ] I checked my UI changes against the design and there are no notable differences
-   [ ] I checked my UI changes for any responsiveness issues
-   [ ] I checked my (code) changes for obvious issues, debug statements and commented code
-   [ ] I provided a screenshot of my changes to the component _(if applicable)_
-   [ ] I regenerated the `icons.html` file and checked if my newly added icon is shown correctly _(if necessary)_
-   [ ] I added an explanation on how to use the component to the readme _(if necessary)_
-   [ ] Documentation _(if necessary)_
-   [ ] Tests _(if necessary)_
-   [ ] Ready to be merged

### CODE REVIEW

File: resources/views/inputs/includes/markdown/navbar.blade.php [hunk 1]
```diff
@@ -101,7 +101,7 @@
                     data-tippy-content="{{ $tooltip ?? trans('ui::markdown.navbar.tooltips.more') }}"
                     data-tippy-offset="[0,-15]"
                 >
-                    <x-ark-icon name="chevron-down" class="inline" size="2xs" />
+                    <x-ark-icon name="arrows.chevron-down-small" class="inline" size="2xs" />
```
Comment (faustbrian) [inline resources/views/inputs/includes/markdown/navbar.blade.php:104]: That doesn't look like PHP 8.1 🤨

File: rector.php [hunk 2]
```diff
@@ -3,179 +3,93 @@
...
+    $services->set(\Rector\Php80\Rector\Class_\ClassPropertyAssignToConstructorPromotionRector::class);
+    $services->set(\Rector\Php80\Rector\Switch_\ChangeSwitchToMatchRector::class);
+    $services->set(\Rector\DeadCode\Rector\StaticCall\RemoveParentCallWithoutParentRector::class);
+    $services->set(\Rector\Php80\Rector\ClassMethod\FinalPrivateToPrivateVisibilityRector::class);
+    $services->set(\Rector\Php80\Rector\ClassMethod\OptionalParametersAfterRequiredRector::class);
+    $services->set(\Rector\Php80\Rector\FuncCall\Php8ResourceReturnToObjectRector::class);
+
+    // php8.1
+    $services->set(\Rector\Php81\Rector\Class_\MyCLabsClassToEnumRector::class);
+    $services->set(\Rector\Php81\Rector\MethodCall\MyCLabsMethodCallToEnumConstRector::class);
```
Comment (faustbrian) [inline rector.php:89]: I don't think we use https://github.com/myclabs/php-enum?

File: rector.php [hunk 3]
```diff
@@ -3,179 +3,93 @@
...
+    $services->set(\Rector\Php80\Rector\Catch_\RemoveUnusedVariableInCatchRector::class);
+    $services->set(\Rector\Php80\Rector\Class_\ClassPropertyAssignToConstructorPromotionRector::class);
+    $services->set(\Rector\Php80\Rector\Switch_\ChangeSwitchToMatchRector::class);
+    $services->set(\Rector\DeadCode\Rector\StaticCall\RemoveParentCallWithoutParentRector::class);
+    $services->set(\Rector\Php80\Rector\ClassMethod\FinalPrivateToPrivateVisibilityRector::class);
+    $services->set(\Rector\Php80\Rector\ClassMethod\OptionalParametersAfterRequiredRector::class);
+    $services->set(\Rector\Php80\Rector\FuncCall\Php8ResourceReturnToObjectRector::class);
+
+    // php8.1
+    $services->set(\Rector\Php81\Rector\Class_\MyCLabsClassToEnumRector::class);
```
Comment (faustbrian) [inline rector.php:88]: I don't think we use https://github.com/myclabs/php-enum?

File: resources/views/inputs/includes/markdown/navbar.blade.php (hunk 1 above)
Comment (leMaur) [reply resources/views/inputs/includes/markdown/navbar.blade.php:104]: ops, wrong PR... :trollface:

**Pull request merged**